spec/support/authentication_helpers.rb

```diff
@@ -0,0 +1,30 @@
+module AuthenticationFeatureHelpers
+  def login_as(user)
+    GDS::SSO.test_user = user
+  end
+
+  def editor_user
+    @editor_user ||= FactoryBot.build(:user)
+  end
+
+  def super_admin_user
+    @super_admin_user ||= FactoryBot.build(:user, :super_admin)
+  end
+
+  def login_as_editor_user
+    login_as editor_user
+  end
+
+  def login_as_super_admin_user
+    login_as super_admin_user
+  end
+end
+
```

Comment:
[Inspired by](https://github.com/alphagov/content-tagger/blob/79047af5d0ac877fc9ad31ea78bc96d823a8eca0/spec/support/authentication_helpers.rb).

I think these are probably handy - there are questions on style and when to use them.

Reply:
I agree, these would be extremely helpful